MathProblems/326_Power_of_Three.py

```
@@ -0,0 +1,18 @@
+class Solution:
+    def isPowerOfThree(self, n: int) -> bool:
+        # by frequently multiplying the number 
+        x=1 
+        while x<n:
+             x*=3 
+             if x==n: return True 
+        return False
+import math
+class Solution:
```

Ruff (0.11.9)

Redefinition of unused `Solution` from line 1

(F811)